python/dask_cudf/dask_cudf/tests/test_distributed.py

```diff
@@ -22,87 +25,107 @@ def at_least_n_gpus(n):
     return ngpus >= n
 
 
+@pytest.fixture
```

Comment:
Like you helped identify with the cudf_polars tests, should this be made module scope to not recreate this per test?